format/index.md

```diff
@@ -1511,8 +1511,8 @@ An error object **MAY** have the following members:
   associated resource(s) in the request document. Each path **MUST** be relative
   to the resource path(s) expressed in the error object's `"links"` member
   [e.g. `["/first-name", "/last-name"]` to reference a couple attributes].
+ * `"meta"` - to contain non-standard meta-information about error.
```

Comment:
The indentation for this item looks off in the rendered markdown. Also it might be better to say `the error` instead of `error`.

Otherwise :+1: